static/templates/settings/dropdown_list_widget.hbs

```diff
@@ -1,10 +1,16 @@
 <div class="input-group dropdown-list-widget" id="{{widget_name}}_widget">
     <span class="{{widget_name}}_setting dropup actual-dropdown-menu prop-element" id="id_{{widget_name}}" aria-labelledby="{{widget_name}}_label">
         {{#if label}}
+        {{#if block}}
+        <label id="{{widget_name}}_label" class="dropdown-title">
+            {{ label }}
+        </label>
+        {{else}}
         <label id="{{widget_name}}_label" class="inline-block">
             {{ label }}
         </label>
```

Comment:
Do we have `dropdown_list_widget` elements that specify `label` and don't want this new behavior? It seems possible we should just be changing the class to `dropdown-title` instead of adding this conditional.

Reply:
I think the answer is no.